### PR TITLE
Show clients status count on menu header

### PR DESCRIPTION
**Pull request by thisajaychauhan**
**Targets #3063** 

### Description
---
To display the count of active and inactive clients on their respective buttons in the client module.

---
**Previous client module UI:-**
235961414-329cf483-6825-4d2c-87ed-8fd7ee8861fb
---
---
**Now client module UI:-**
Screenshot (103)
---
 
### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Client/Resources/views/menu_header.blade.php [hunk 1]
```diff
@@ -6,13 +6,13 @@
         @php
             $request['status'] = 'active';
         @endphp
-        <a class="nav-link {{ (request()->input('status', 'active') == 'active')  ? 'active' : '' }}" href="{{ route('client.index', $request)  }}">Active Clients</a>
+        <a class="nav-link {{ (request()->input('status', 'active') == 'active')  ? 'active' : '' }}" href="{{ route('client.index', $request)  }}">Active Clients {{ '('.$activeCount.')' }}</a>
```
Comment (Ayush8923) [inline Modules/Client/Resources/views/menu_header.blade.php:9]:
```suggestion
        <a class="nav-link {{ (request()->input('status', 'active') == 'active')  ? 'active' : '' }}" href="{{ route('client.index', $request)  }}">Active Clients ({{$activeCount}})</a>
```

File: Modules/Client/Resources/views/menu_header.blade.php [hunk 2]
```diff
@@ -6,13 +6,13 @@
...
     </li>
 
     <li class="nav-item">
         @php
             $request['status'] = 'inactive';
         @endphp
-        <a class="nav-link {{ (request()->input('status', 'active') == 'inactive') ? 'active' : '' }}"  href="{{ route('client.index', $request)  }}">Inactive Clients</a>
+        <a class="nav-link {{ (request()->input('status', 'active') == 'inactive') ? 'active' : '' }}"  href="{{ route('client.index', $request)  }}">Inactive Clients {{ '('.$inactiveCount.')' }}</a>
```
Comment (Ayush8923) [inline Modules/Client/Resources/views/menu_header.blade.php:16]:
```suggestion
        <a class="nav-link {{ (request()->input('status', 'active') == 'inactive') ? 'active' : '' }}"  href="{{ route('client.index', $request)  }}">Inactive Clients ({{$inactiveCount}})</a>
```

File: Modules/Client/Services/ClientService.php [hunk 3]
```diff
@@ -43,7 +43,10 @@ public function index(array $data = [])
             $clients = $clients->diff($client->linkedAsDepartment);
         }
 
-        return ['clients' => $clients, 'count' => $count];
+        $activeCount = Client::where('status', 'active')->count();
```
Comment (Ayush8923) [inline Modules/Client/Services/ClientService.php:46]:
```suggestion
        $activeClientsCount = Client::where('status', 'active')->count();
```

File: Modules/Client/Services/ClientService.php [hunk 4]
```diff
@@ -43,7 +43,10 @@ public function index(array $data = [])
...
+        $inactiveCount = Client::where('status', 'inactive')->count();
```
Comment (Ayush8923) [inline Modules/Client/Services/ClientService.php:47]:
```suggestion
        $inactiveClientsCount = Client::where('status', 'inactive')->count();
```

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Left some comments. Please check.

File: Modules/Client/Resources/views/menu_header.blade.php (hunk 2 above)
Comment (thisajaychauhan) [reply Modules/Client/Resources/views/menu_header.blade.php:16]: done.

File: Modules/Client/Services/ClientService.php (hunk 4 above)
Comment (thisajaychauhan) [reply Modules/Client/Services/ClientService.php:47]: done.

**Review by Ayush8923 — APPROVED** [review]

**Pull request merged**